src/templates/app/user/security/account_closure.jsx

```diff
@@ -32,7 +32,29 @@ const ClosureDescription = ({
         </div>
     </React.Fragment>
 );
-    
+ 
+const AccountClosureDialog = () => (
+    <div id='account-closure-dialog' className='lightbox'>
+        <div id='account-closure-dialog-content' className='gr-padding-10 gr-gutter'>
+            <div className='center-text gr-padding-10'>
+                <img
+                    id='ic-emergency'
+                    className='responsive'
+                    src={it.url_for('images/pages/account_closure/ic-emergency.svg')}
+                />
+                <h1 className='gr-padding-10'>{it.L('Warning!')}</h1>
+                <p className='warning-msg'>{it.L('If you deactivate:')}</p>
+                <p className='warning-msg'>{it.L('You’ll be logged out automatically.')}</p>
+                <p className='warning-msg'>{it.L('You will ')}<span id='red-msg'>{it.L('NOT')}</span>{it.L(' be able to log in again.')}</p>
```

Comment:
Better to use `placeholders` here and not to split the sentence.
You can use:
`it.L('You will [_1] be able to log in again.', `&lt;span id='red-msg'&gt;${it.L('NOT')}&lt;/span&gt;`)`

Reply:
no, this should be `it.L('You will [_1]NOT[_2] be able to log in again.', '<span id="red-msg">', '</span>')` so that translators can see the whole string and structure the grammar however they need to

Reply:
Right :+1: